pymeasure/display/widgets/results_dialog.py

```diff
@@ -37,6 +37,14 @@
 
 
 class ResultsDialog(QtGui.QFileDialog):
+    """
+    Widget that displays a dialog box for loading a past experiment run.
+    It shows a preview of curves from the results file when selected in the dialog box.
+
+    This widget is implemented in the `open_experiment` method in
```

Comment:
Maybe rather "This widget is **used by** the `open_experiment` method in"

Reply:
I pushed up that change.

And to the CI test, it just failed in the same pytest for workers, at a different line:
`tests/experiment/test_workers.py:111: AssertionError`